nifi-extension-bundles/nifi-poi-bundle/nifi-poi-services/src/test/java/org/apache/nifi/processors/excel/TestSplitExcel.java

```diff
@@ -239,6 +241,38 @@ void testCopyDateTime() throws Exception {
         }
     }
 
+    @Test
+    void testHyperlinks() throws IOException {
+        final Path hyperlinksFile = Paths.get("src/test/resources/excel/hyperlinks.xlsx");
+        runner.enqueue(hyperlinksFile);
+
+        runner.run();
+
+        runner.assertTransferCount(SplitExcel.REL_SPLIT, 1);
+        runner.assertTransferCount(SplitExcel.REL_ORIGINAL, 1);
+        runner.assertTransferCount(SplitExcel.REL_FAILURE, 0);
+
+        final MockFlowFile flowFile = runner.getFlowFilesForRelationship(SplitExcel.REL_SPLIT).getFirst();
+        try (XSSFWorkbook workbook = new XSSFWorkbook(flowFile.getContentStream())) {
+            final Sheet sheet = workbook.getSheetAt(0);
+            assertEquals("Sheet1", sheet.getSheetName());
+
+            final List<XSSFHyperlink> hyperlinks = (List<XSSFHyperlink>) sheet.getHyperlinkList();
+            Assertions.assertIterableEquals(
```

Comment:
The `import static` approach should be used instead of `Assertions`:
```suggestion
            assertIterableEquals(
```

Reply:
Done